tests/test_hf_api.py

```diff
@@ -139,6 +139,25 @@ def test_login_cli(self):
             read_from_credential_store(USERNAME_PLACEHOLDER), (None, None)
         )
 
+    def test_login_deprecation_error(self):
+        with pytest.warns(
+            FutureWarning,
+            match=r"HfApi.login: This method is deprecated in favor of "
+            r"`set_access_token` and will be removed in v0.7.",
```

Comment:
I think this is not black formatted.

Reply:
Actually, it is! I've split it in two to respect the 88 line char limit, black seems okay with it

Reply:
Black unindented it, the second line was previously indented at the same level as the `|` in `match=|`

Reply:
OK